PSSSRS/public-functions/Get-RSConfigurationSetting.ps1

```diff
@@ -0,0 +1,86 @@
+<#
+Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+Licensed under the MIT License (MIT)
+#>
+
+function Get-RSConfigurationSetting
+{
+<#
+.SYNOPSIS
+
+.EXAMPLE
+New-RSConfigurationSession -ComputerName server

```

Comment:
the example is for +New-RSConfigurationSession, should be updated to Get-RSConfigurationSetting